src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -531,20 +531,22 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
                 }
             })?;
 
-        self.resolve_path(&path_root, TypeNS, module_id).and_then(|ty_res| {
-            self.resolve_associated_item(ty_res, item_name, ns, module_id, extra_fragment)
-        }).unwrap_or_else(|| {
-            if ns == Namespace::ValueNS {
-                self.variant_field(path_str, module_id)
-            } else {
-                Err(ResolutionFailure::NotResolved {
-                    module_id,
-                    partial_res: None,
-                    unresolved: path_root.into(),
+        self.resolve_path(&path_root, TypeNS, module_id)
```

Comment:
I don't see what changed here. Won't rustfmt just undo this? (If not, why not?) Or is this (and the block starting with L672) actually caused by rustfmt?